nautobot/core/middleware.py

```diff
@@ -22,14 +34,38 @@ def header(self):
         return settings.REMOTE_AUTH_HEADER
 
     def process_request(self, request):
-
         # Bypass middleware if remote authentication is not enabled
-        if not settings.REMOTE_AUTH_ENABLED:
+        if not remote_auth_enabled(auth_backends=settings.AUTHENTICATION_BACKENDS):
```

Comment:
I'm curious - is there a technical reason that we have to pass `settings.AUTHENTICATION_BACKENDS` as a parameter when calling the `*_auth_enabled()` functions, or would it be possible (or even better?) to make them parameter-less, i.e. have them pull in `settings.AUTHENTICATION_BACKENDS` automatically?

Reply:
There is a technical reason for this approach.

Reply:
We won't want to have a circular dependency between settings funcs and settings.